app/controllers/ReviewDetailsController.scala

```diff
@@ -53,16 +56,16 @@ class ReviewDetailsController @Inject()(val authConnector: AuthConnector,
         case Some(businessDetails) =>
           currentBackLink.map(backLink =>
             if (authContext.isAgent && businessDetails.isBusinessDetailsEditable) {
-              Ok(views.html.review_details_non_uk_agent(serviceName, businessDetails, backLink))
+              Ok(template(serviceName, businessDetails, backLink))
```

Comment:
would it better to be more specific about these template names ? rather than template, template2 etc

Reply:
Addressed